cluster/gce/gci/configure-helper.sh

```diff
@@ -682,15 +685,15 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
+      flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
```

Comment:
Is this flag necessary? It wasn't passed in before and as long as the kubeconfig file is at the default location the kubelet should pick it up.